libs/ui-toolkit/src/utils/intent.tsx

```diff
@@ -1,19 +1,42 @@
 export enum Intent {
   None,
+  Danger,
   Primary,
-  Success,
   Warning,
-  Danger,
+  Selected,
```

Comment:
You can also remove the shadow-${intent} and border-${intent} classes

Reply:
@mattrussell36 Sounds good to me, I'll check that Pete and Michael are happy too. I'd like to avoid spec creep in PRs though, so will do this as a separate ticket.